pkg/decision/evaluator/condition_test.go

```diff
@@ -81,3 +81,24 @@ func TestCustomAttributeConditionEvaluatorWithoutMatchType(t *testing.T) {
 	result, _ = conditionEvaluator.Evaluate(condition, condTreeParams)
 	assert.Equal(t, result, false)
 }
+
+func TestCustomAttributeConditionEvaluatorForSemver(t *testing.T) {
```

Comment:
Please add GE or add comments in the function, what SemVer test we are asserting.